### ISSUE TITLE
Get my Status

### ISSUE THREAD

**Issue by Bauepete** [jfuerlinger/serta#3]
## Story
As a player I want to get my current status in order to see my progress.

## Example
Player @jfuerlinger is on level M. 
```
+serta-status
```
Expected response 
```
Your Status:

You are in level M. Hospitals line up to get your help!
Immune System: 40 %
```

## DoD
- Calls of `+serta-status @<some name>` list the status of player <some name>
- If the player has an immune system level to be promoted for a next level a message `Your immune system looks good. Address a virologist to get a promotion for the next level`

**Comment by jfuerlinger** [comment]
@Bauepete would you be so kind to provide me some funny messages for the different levels:

e.g. "M" => Hospitals line up to get your help!

It would be great to have more than one message per level.

**Comment by Bauepete** [comment]
Yes this is an issue with low priority (see issue #12). I will add them as we go. Did not have the creativity last night …
